generators/app/tasks/appSetup/baseFilesTemplate.js

```diff
@@ -47,7 +47,8 @@ const {
   I18NEXT_MOCK,
   TESTS_STORE,
   TESTS_SETUP_ENZYME,
-  TESTS_RESPONSES_EXAMPLES
+  TESTS_RESPONSES_EXAMPLES,
+  INITIAL_LOADING
```

Comment:
Nothing change there

Reply:
That is fine

Reply:
yep, you moved `INITIAL_LOADING` to this array, it was in a login file before. just want to know why

Reply:
Because the INITIAL_LOADING not depends from login anymore. Now it will a template, now is an ejs file. And it could change depending on the features of login and onboarding but it will always exist